ui/tests/integration/components/database-role-edit-test.js

```diff
@@ -97,11 +98,16 @@ module('Integration | Component | database-role-edit', function (hooks) {
     await render(hbs`<DatabaseRoleEdit @model={{this.modelStatic}} @mode="create"/>`);
     await fillIn('[data-test-ttl-value="Rotation period"]', '2');
     await click('[data-test-toggle-input="toggle-skip_import_rotation"]');
+    await fillIn('[data-test-input="password"]', 'testPassword'); // fill in password field
 
     await click('[data-test-secret-save]');
 
     await render(hbs`<DatabaseRoleEdit @model={{this.modelStatic}} @mode="show"/>`);
     assert.dom('[data-test-value-div="Rotate immediately"]').containsText('No');
+    assert.dom('[data-test-value="Password"]').doesNotExist(); // verify password field doesn't show on details view
```

Comment:
Same comment here, if these selectors already exists in general-selectors, replace them with that pattern.